### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by sheylaPozo**
# Set up associations

- Create a class Classroom
- Create the many belongs-to relationships between Classroom and Student.
- Create a class Book.
- Create a class Rental.

Thank you for your kind review✨😊

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```
Comment (ElijahTobs) [inline book.rb:4]: - [x] In order to properly show the `has-many` relationship between book and rental, getters and setters for rentals should be present.

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
```
Comment (ElijahTobs) [inline rental.rb:4]: - [x] Getters and Setters for @book and @person instance variables should be present

File: person.rb [hunk 3]
```diff
@@ -10,6 +10,7 @@ def initialize(params)
     @name = validate_name params[:name] || 'Unknown'
     @age = params[:age]
     @parent_permission = params.fetch(:parent_permission, true)
+    @rentals = []
   end
 
   def can_use_services?
```
Comment (ElijahTobs) [inline person.rb]: - [x] In order to properly show the `has-many` relationship between person and rental, getters and setters for rentals should be present.

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @sheylaPozo  👋

You've done really well so far! 👍 

## Major Highlights 💯 

Your README is professional ✔️
You've followed the Gitflow rules. ✔️ 
Nice PR title and summary✔️ 
No linter errors✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

_See the inline comments below_


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: book.rb (hunk 1 above)
Comment (sheylaPozo) [reply book.rb:4]: ok, completed.

File: rental.rb (hunk 2 above)
Comment (sheylaPozo) [reply rental.rb:4]: OK, completed.

File: person.rb (hunk 3 above)
Comment (sheylaPozo) [reply person.rb]: fixed it

**Review by AshakaE — APPROVED** [review]
### Hi @sheylaPozo  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the requested changes 👏🏽
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sheylaPozo** [comment]
thank you :)

**Pull request merged**